### PR TITLE
Create UKCore-Extension-SampleCategory-PheobeSmitham-Example.xml

### PR DESCRIPTION
**Pull request by vickyjaiswal0308**

### CODE REVIEW

**Review by KazeemHamzat — CHANGES_REQUESTED** [review]
For consistency, would you want to convert the content to XML? The file extension is XML but content is in JSon format.

**Comment by vickyjaiswal0308** [comment]
> For consistency, would you want to convert the content to XML? The file extension is XML but content is in JSon format.

Thanks Kazeem , I think in a rush I had committed it without converting the file into xml. I have change the file to xml now.

File: examples/UKCore-Extension-SampleCategory-PheobeSmitham-Example.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<Specimen xmlns='http://hl7.org/fhir'>
+  <id value='Specimen-PheobeSmitham-Example'/>
```
Comment (KazeemHamzat) [inline examples/UKCore-Extension-SampleCategory-PheobeSmitham-Example.xml:2]: Id does not conform to file name as expected in UKCore Design guide.
@ryma2fhir Given that this resource also exist in Genomics IG with id: Specimen-PheobeSmitham-Example. Would you give your opinion on this?

Comment (ryma2fhir) [reply examples/UKCore-Extension-SampleCategory-PheobeSmitham-Example.xml:2]: I think the best way would be to add it to UK Core as `UKCore-Specimen-PheobeSmitham-Example` and if it's the same example in genomics, could we render the example from UK Core package into the genomics IG?

Comment (vickyjaiswal0308) [reply examples/UKCore-Extension-SampleCategory-PheobeSmitham-Example.xml:2]: I can see there's already an example with the Id : UKCore-Extension-SampleCategory-Example . And as per the Quality Control checker if i'll give the id value as : UKCore-Extension-SampleCategory-PheobeSmitham-Example seems to be a wrong format. So, should i give the id value as something like UKCore-Extension-SampleCategory-Example2? @ryma2fhir can you give your suggestion?

Comment (ryma2fhir) [reply examples/UKCore-Extension-SampleCategory-PheobeSmitham-Example.xml:2]: We have done this before, so lets go with `UKCore-Extension-SampleCategory-PheobeSmitham-Example`
https://github.com/NHSDigital/FHIR-R4-UKCORE-STAGING-MAIN/blob/develop/examples/UKCore-Extension-PriorityReason-SendingAsText-Example.xml

**Review by KazeemHamzat — APPROVED** [review]
All looks good for me.

**Pull request merged**